content/en/agent/guide/agent-configuration-files.md

```diff
@@ -74,28 +74,24 @@ Prior releases of Datadog Agent stored configuration files in `/dd-agent/conf.d/
 
 ### Checks configuration files for Agent 6
 
-In order to provide a more flexible way to define the configuration for a check, from version 6.0.0, the Agent loads any valid YAML file contained in the folder:
-
-`/etc/datadog-agent/conf.d/<CHECK_NAME>.d/`.
-
-Note: For log collection, to prevent duplicate logs from being sent to Datadog, the Agent does not accept multiple YAML files that point to the same log source. In the case where there is more than one YAML file that points to the same log source, the Agent considers the files in alphabetical order and uses the first file.
-
-This way, complex configurations can be broken down into multiple files. For example, a configuration for the `http_check` might look like this:
+An example for each Agent check configuration files can be found in the `conf.yaml.example` file in the corresponding `<CHECK_NAME>.d/` folder. Rename this file to `conf.yaml` in order to enable the associated check. Note that the Agent loads any valid YAML file contained in the folder: `/etc/datadog-agent/conf.d/<CHECK_NAME>.d/`. This way, complex configurations can be broken down into multiple files. For example, a configuration for the `http_check` might look like this:
 
 ```
 /etc/datadog-agent/conf.d/http_check.d/
 ├── backend.yaml
 └── frontend.yaml
 ```
 
-Autodiscovery template files are stored in the configuration folder as well. For example, consider `redisdb`:
+Autodiscovery template files are stored in the configuration folder as well as a `auto_conf.yaml` file. For example, consider the Redis check, its configuration in `redisdb.d/` are:
 
 ```
 /etc/datadog-agent/conf.d/redisdb.d/
 ├── auto_conf.yaml
 └── conf.yaml.example
 ```
 
+**Note**: For log collection, to prevent duplicate logs from being sent to Datadog, the Agent does not accept multiple YAML files that point to the same log source. In the case where there is more than one YAML file that points to the same log source, the Agent considers the files in alphabetical order and uses the first file.
```

Comment:
```suggestion
**Note**: For log collection, the Agent does not accept multiple YAML files that point to the same log source  to prevent duplicate logs from being sent to Datadog. In the case where there is more than one YAML file that points to the same log source, the Agent considers the files in alphabetical order and uses the first file.
```